src/console/Command.hpp

```diff
@@ -0,0 +1,60 @@
+#ifndef CONSOLE_COMMAND_HPP
+#define CONSOLE_COMMAND_HPP
+
+#include "storm/Hash.hpp"
```

Comment:
For external libraries like `storm`, prefer angle brackets: `#include <storm/Hash.hpp>`